quinn-proto/src/connection/mod.rs

```diff
@@ -2943,6 +2994,22 @@ where
         self.path.sending_ecn
     }
 
+    /// Instruct the peer to retire the next `n` connection IDs, and issue replacements
+    #[cfg(test)]
+    pub(crate) fn rotate_rem_cid(&mut self, v: u64, n: u64) {
+        // Cannot retire more CIDs than what have been issued
+        debug_assert!(v <= *self.cids_active_seq.iter().max().unwrap() + 1);
+        self.retire_cid_seq = v;
+        self.endpoint_events
+            .push_back(EndpointEventInner::NeedIdentifiers(n));
+    }
```

Comment:
Do we actually need to have two separate arguments to this, or would a single quantity of CIDs to both retire and issue be sufficient?

Reply:
Since it is a test function, i don't have a strong preference here. The intention was to issue less new CIDs than retired ones, which makes it easy to read the log. Anyway, i am happy to update it.